pandas_gbq/tests/test_gbq.py

```diff
@@ -19,9 +19,9 @@
 import pandas.util.testing as tm
 from pandas.compat.numpy import np_datetime64_compat
 
-PROJECT_ID = None
-PRIVATE_KEY_JSON_PATH = None
-PRIVATE_KEY_JSON_CONTENTS = None
+PROJECT_ID = os.getenv('GOOGLE_CLOUD_PROJECT')
```

Comment:
I am ok with these environment variables (They are just for local testing). I agree with @jreback that is also necessary to follow the steps in the contributing docs to configure Travis-CI to run on your pandas-gbq fork. You only need to configure Travis-CI once. After that unit tests run automatically for your pandas-gbq fork after each push.